src/docker-compose-2-clients.yml

```diff
@@ -0,0 +1,125 @@
+#
+# Copyright (C) 2024 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+ ## Current Developer Toolbox doesn't support environment files, make sure to remove any files or environment variables starting with $
+version: '3.7'
+services:
+  camera-simulator:
+    container_name: camera-simulator
+    image: aler9/rtsp-simple-server
+    ports:
+      - "127.0.0.1:8554:8554"
+  camera-simulator0:
+    image: jrottenberg/ffmpeg:4.1-alpine
+    container_name: camera-simulator0
+    network_mode: "host"
+    entrypoint: ["/bin/sh","-c"]
+    command: 
+    - | 
+       if [ ! -f /home/pipeline-server/sample-media/coca-cola-4465029-1920-15-bench.mp4 ]; then
+          mkdir -p /home/pipeline-server/sample-media
+          wget -O /home/pipeline-server/sample-media/coca-cola-4465029-1920-15-bench.mp4 https://www.pexels.com/download/video/4465029
+       fi
+       ffmpeg -nostdin -re -stream_loop -1 -i /home/pipeline-server/sample-media/coca-cola-4465029-1920-15-bench.mp4 -c copy -f rtsp -rtsp_transport tcp rtsp://localhost:8554/camera_0
+    depends_on:
+      - camera-simulator
+    volumes:
+      - ${RETAIL_USE_CASE_ROOT:-..}/performance-tools/sample-media:/home/pipeline-server/sample-media
+
+  OvmsClientGst1:
```

Comment:
This container isn't OpenVINO Model Server, we should change the name of it. Maybe `GstClient`

Reply:
fixed.